src/main/java/seedu/address/logic/parser/DeleteAppointmentCommandParser.java

```diff
@@ -3,27 +3,26 @@
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
 import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.DeleteAppointmentCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new DeleteCommand object
+ * Parses input arguments and creates a new DeleteAppointmentCommand object
  */
-public class DeleteCommandParser implements Parser<DeleteCommand> {
+public class DeleteAppointmentCommandParser {
```

Comment:
this doesn't look like it fits the regular format of implementing `Parser<DeleteAppointmentCommand>`